frontend/lib/src/components/shared/StreamlitMarkdown/StreamlitMarkdown.tsx

```diff
@@ -92,7 +92,7 @@ export interface Props {
   largerLabel?: boolean
 
   /**
-   * Does not allow links & has larger font sizing
+   * Does not allow links
    */
   isButton?: boolean
```

Comment:
Previously `isButton` handled both disabling links and larger label behavior - separated this logic for clarity & for radio captions use case which disables links but maintains smaller labels